src/console/controllers/ServeController.php

```diff
@@ -28,6 +28,12 @@ class ServeController extends BaseServeController
      */
     public $docroot = '@webroot';
 
+    /**
+     * @var string path or [path alias](guide:concept-aliases) to router script.
+     * See https://www.php.net/manual/en/features.commandline.webserver.php
+     */
+    public $router = '@app/../bootstrap/router.php';
```

Comment:
One questions someone with more domain knowledge may be able to answer. I wasn't sure if there was a better alias than `@app` so I didn't need the `../`?

Reply:
I don't think there's anything else you could use.

Reply:
`@vendor/craftcms/cms` might feel a bit cleaner